tools/apprunner-maven-plugin/src/main/java/com/dremio/nessie/quarkus/maven/QuarkusAppStartMojo.java

```diff
@@ -70,6 +73,12 @@
   @Parameter(property = "nessie.apprunner.appArtifactId", required = true)
   private String appArtifactId;
 
+  /**
+   * Application configuration properties.
+   */
+  @Parameter
```

Comment:
This is used via a `<properties>` block in the config of the plugin then? And its a user error if the properties are invalid or contracticting

Reply:
the configuration key is derived from the field name, so in this case, it's `applicationProperties`. They take precedence over values defined in `application.properties`